src/screens/scan-qr-code/scan-qr-code.tsx

```diff
@@ -75,6 +86,11 @@ export const ScanQrCode = () => {
         notAuthorizedView={<EmptyQrCode />}
         onRead={handleRead}
       />
+      {isCameraPermissionGranted === false && isAndroid && (
```

Comment:
rename to `isAndroidCameraPermissionGranted` and remove isAndroid check (you do that check inside useEffect)